quixstreams/models/serializers/quix.py

```diff
@@ -221,6 +224,11 @@ def __call__(self, value: bytes, ctx: SerializationContext) -> Iterable[dict]:
                 f'Ignore Quix message with "{QModelKey.HEADER_NAME}": "{model_key}"'
             )
 
+        # Check if compressed
+        if codec_id.startswith(QCodecId.GZIP_PREFIX):
+            codec_id = codec_id[len(QCodecId.GZIP_PREFIX) :]
+            value = gzip.decompress(value)
+
```

Comment:
```suggestion
            if not (
                codec_id.startswith(QCodecId.GZIP_PREFIX) 
                and (codec_id := codec_id[len(QCodecId.GZIP_PREFIX) :]) in QCodecId.SUPPORTED_CODECS
            ):
                raise SerializationError(
                    f'Unsupported "{QCodecId.HEADER_NAME}" value "{codec_id}"'
                )
            value = gzip.decompress(value)
```

Reply:
This suggestion closely matches the behavior you had before; basically if you had `[GZIP]badcodec`, would raise `'... value "badcodec"'`, while also raising `..value "[NOTGZIP]goodcodec"` if it didn't start with GZIP